src/test/java/delete_task_use_case/DeleteTaskControllerTest.java

```diff
@@ -0,0 +1,36 @@
+package delete_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.DataAccessTask;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeleteTaskControllerTest {
+    private final static DeleteTaskPresenter outputBoundary = new DeleteTaskPresenter();
+    private final static DeleteTaskDataAccess dataAccess = new DeleteTaskDataAccess();
```

Comment:
Again, this should be a DsGateway instance since that is how the application logic is executed in the actual program.

Reply:
That is true, changes added.